### DISCUSSION TITLE
Which AI logo for csv,conf,v8 should I use?

### DISCUSSION THREAD

**Discussion by jqnatividad** [dathere/qsv#1843] [Polls]
To break the ice during my talk at csv,conf,v8-2024, I decided to create fun, conference-themed logos using ChatGPT 4o, with [Speedy Gonzales](https://en.wikipedia.org/wiki/Speedy_Gonzales) - one of my Saturday morning cartoon childhood favorites, as an inspiration.

These logos were generated by providing the original qsv logo to ChatGPT 4o and asking it to generate a new logo based on the original. The results are quite interesting and were generated using the following prompt sequence:

<table>
<tr>
<td><img src="https://raw.githubusercontent.com/jqnatividad/qsv/master/docs/images/qsv-logo.png" width="200"></td>
<td valign="top"><b>Original qsv logo</b>: <br><br>  > Can you modify the logo so the rider looks like Speedy Gonzales?</td>
</tr>
<tr>
<td width="230"><img src="https://raw.githubusercontent.com/jqnatividad/qsv/master/docs/images/ckanconfv8-2024/qsv-ai-logo1.png"  width="200" height="200"></td>
<td valign="top"><b>Logo 1: "Realistic"</b><br><br>  > Keep the general outline of the logo. Also, make it look like the cartoon character "Speedy Gonzales"</td>
</tr>
<tr>
<td><img src="https://raw.githubusercontent.com/jqnatividad/qsv/master/docs/images/ckanconfv8-2024/qsv-ai-logo2.png"  width="200" height="200"></td>
<td valign="top"><b>Logo 2: "Ladies' Man"</b><br><br> > Please create another version that looks more like the original. This time, only change the rider so it looks like the Looney Tunes cartoon character "Speedy Gonzales"</td>
</tr>
<tr>
<td><img src="https://raw.githubusercontent.com/jqnatividad/qsv/master/docs/images/ckanconfv8-2024/qsv-ai-logo3.png"  width="200" height="200"></td>
<td valign="top"><b>Logo 3: "BigHat"</b><br><br> > Nice! Can you correct the spelling of "CSSV DATAWANGLLER" and change it to "CSV DATAWRANGLER"?</td>
</tr>
<tr>
<td><img src="https://raw.githubusercontent.com/jqnatividad/qsv/master/docs/images/ckanconfv8-2024/qsv-ai-logo4.png"  width="200" height="200"></td>
<td valign="top"><b>Logo 4: "Angry Data Wanglers"</b><br><br>  > It's still misspelled!<br><br><br>  
Somehow, it generated two logos for the last prompt. It first generated Logo 5, and as I was typing that it was still misspelled, it generated Logo 6 unprompted!</td>
</tr>
<tr>
<td><img src="https://raw.githubusercontent.com/jqnatividad/qsv/master/docs/images/ckanconfv8-2024/qsv-ai-logo5.png"  width="200" height="200"></td>
<td valign="top"><b>Logo 5: "Zorro-tic"</b></td>
</tr>
<tr>
 <td><img src="https://raw.githubusercontent.com/jqnatividad/qsv/master/docs/images/ckanconfv8-2024/qsv-ai-logo6.png"  width="200" height="200"></td>
<td valign="top"><b>Logo 6: "The Rock Wrangler"</b></td>
</tr>
</table> 

---

These LLMs are amazing! ...and kinda scary at the same time... My favorites are "Big Hat" and "Zorro-tic".

**Comment by ondohotola** [comment]
I would on each try to make the lasso show the QSV similar to the original. And of course correct the spelling errors

**Reply by ondohotola** [reply]
Interesting, I looked up Speedy Gonzales on Google and looked at a short You Tube clip. I am now inundated by Looney Tunes on my Facebook feed. Nothing to do with QSV of course :-)-O

**Reply by jqnatividad** [reply]
It's only really an ice-breaker for the conference, and for me to get to mention obligatory "AI" in my talk.

The intentionally kitschy original logo will remain for now... until AI models can do the qsv lasso treatment and take spelling instructions, that is. 😉

**Comment by jqnatividad** [comment]
I ended up using them all :smile:

The slides are now available at https://github.com/jqnatividad/qsv/blob/master/docs/slides/CSVconf-2024.pdf
